dhis-2/dhis-services/dhis-service-field-filtering/src/main/java/org/hisp/dhis/fieldfiltering/FieldFilterService.java

```diff
@@ -76,7 +75,7 @@
 /**
  * @author Morten Olav Hansen
  */
-@Service
+@Service( "org.hisp.dhis.fieldfiltering.FieldFilterService" )
```

Comment:
Do we need the name? We try to get away from explicit names.

Reply:
I thought this is our convention as per https://github.com/dhis2/wow-backend/blob/master/guides/spring_conventions.md#bean-id

Now that I re-read it I see this one is not implementing an interface 😅 I'll remove that change 👌🏻